### PR TITLE
Port libs from Cockpit

### PR DESCRIPTION
**Pull request by marusak**
@garrett can I get your review and help please?

We cannot switch to two columns, see the tmp commit. But then the buttons are not align, see:
buttonscommon

Also other visual change is that `Get new image` is aligned bit differently, is this correct? Otherwise there are not obvious changes.

### CODE REVIEW

File: lib/form-layout.less [hunk 1]
```diff
@@ -63,28 +63,37 @@
...
   grid-gap: @padding-small-vertical @padding-small-horizontal;
   // Repeat a label that is a minimum of 4em and its control that
   // fills the remaining space by a CSS variable (default: 2)
   grid-template-columns: repeat(var(--ct-form-columns), max-content 1fr);
-  place-items: baseline stretch;
+  justify-items: stretch;
+  align-content: baseline;
 
-  // All <label> elements describing form elements in PatternFly are
+  // All <label>gg/uu elements describing form elements in PatternFly are
```
Comment (garrett) [inline lib/form-layout.less]: I have fix for this comment @ #12696 :wink:

It's commit https://github.com/cockpit-project/cockpit/pull/12696/commits/0b0eaced57efb9b22b5b58a950400607de9e1aa4

**Comment by garrett** [comment]
1. fixed alignment in 025b47f
2. ported https://github.com/cockpit-project/cockpit/commit/0b0eaced57efb9b22b5b58a950400607de9e1aa4 here as e644774

**Comment by marusak** [comment]
Thank you @garrett !
I squashed it together, could you please formally approve?

**Comment by marusak** [comment]
> Can you please re-push with a Closes #

Ach, shame, sorry, I did a few force pushes but forgot to add closes. Fixed.

> This also looks like we should do the eslint bump on cockpit-podman

Yeah, but maybe after we land current PRs :)

**Pull request merged**